tests/compile/test_fusion_attn.py

```diff
@@ -171,6 +179,8 @@ def __init__(self, num_qo_heads: int, num_kv_heads: int, head_size: int,
             cache_config=vllm_config.cache_config,
             prefix="model.layers.0.self_attn.attn",
         )
+        self.attn._k_scale = self.attn._k_scale.to(device)
+        self.attn._v_scale = self.attn._v_scale.to(device)
```

Comment:
Why is this necessary? Where would ROCm actually do this? Because I think it might break the Blackwell FI?

Reply:
Why not just set the default device at the start of the test then?